src/arrayMethodReduce.js

```diff
@@ -6,6 +6,19 @@
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
     // write code here
+    let prev = initialValue;
```

Comment:
What is `prev`? Better name it as `accumulator`.

Reply:
> What is `prev`? Better name it as `accumulator`.

Thank you for the advice, I changed it